persimmon/website/templates/website/login.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
```

Comment:
This should extend the main template instead of building its own boilerplate html

Reply:
this was a accidental upload